pints/_log_likelihoods.py

```diff
@@ -578,57 +578,78 @@ class MultiplicativeGaussianLogLikelihood(pints.ProblemLogLikelihood):
     .. math::
         v(t) \sim \text{ iid } N(0, \sigma)
 
-    Note that the scalar parameter ``eta`` controls the exponential dependence
-    of the noise on the function output, while the scalar parameter ``sigma``
-    provides a baseline level of the noise standard deviation. This model leads
-    to a log likelihood of
+    Note that the scalar parameter :math:`\eta` controls the exponential
```

Comment:
I think f and X need to be introduced here

Reply:
Also f(t, \theta) instead of f(t; \theta) (twice)

Reply:
Done